src/cmd/services/m3coordinator/downsample/flush_handler.go

```diff
@@ -120,11 +120,11 @@ func (w *downsamplerFlushHandlerWriter) Write(
 		tags := make(models.Tags, 0, expected+1)
```

Comment:
nit: This should actually just be `expected` not `expected+1` since we increment `expected` above if there is an aggregation tag.